.github/workflows/generator-test.yml

```diff
@@ -6,11 +6,18 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v4
-      - uses: devcontainers/ci@v0.3
```

Comment:
Ahaha 😅 I forgot to add Java install stage to the Dockerfile, not sure why.

The one action that has succeded has it.
https://github.com/deirn/fabricmc.net/blob/1d9933057431acb613e6a2601519979d0fc9fc37/Dockerfile#L15-L17

Reply:
Oh thats clever, I wasnt sure how to install Java in an existing docker container. We should prob move the main build over to the devcontainer. We likely specify the node version in about 10 places 😆